api.planx.uk/helpers.ts

```diff
@@ -4,15 +4,32 @@ import { Flow, Node } from "./types";
 import { ComponentType, FlowGraph } from "@opensystemslab/planx-core/types";
 import { $public, getClient } from "./client";
 
+export interface FlowData {
+  slug: string;
+  data: Flow["data"];
```

Comment:
This, like many other helper functions here, are still using outdated `Flow` & `Node` types from our API definitions rather than planx-core's `FlowGraph` & `Node`. I'll plan to pick up this type migration in a separate PR as it's going to introduce a fair bit of noise, and possibly require an update to the `Node` currently defined in planx-core.